lxc/list.go

```diff
@@ -781,3 +828,43 @@ func (c *cmdList) NumberOfProcessesColumnData(cInfo api.InstanceFull) string {
 func (c *cmdList) locationColumnData(cInfo api.InstanceFull) string {
 	return cInfo.Location
 }
+
+func matchByType(cInfo *api.Instance, query string) bool {
+	return strings.ToLower(cInfo.Type) == strings.ToLower(query)
+}
+
+func matchByStatus(cInfo *api.Instance, query string) bool {
```

Comment:
Would it be possible to extend this for using `r,s and f` in addition to `running, stopped, frozen`?
That would we very convenient, but it is not a must-have of course.
I assume it would be changed somewhere else, but I wanted to request it anyway.

Reply:
Hmm, no, we shouldn't do that. We technically have things like `restarting`, `stopping`, `freezing`, ... as valid states which you almost never see (too quick) but should we change that, things would become ambiguous.